services/src/main/java/org/jboss/windup/web/services/service/MigrationProjectService.java

```diff
@@ -68,9 +69,9 @@ public MigrationProject getMigrationProject(Long id)
     {
         String query = "SELECT ctxt FROM AnalysisContext ctxt WHERE ctxt.migrationProject = :project";
 
-        return this.entityManager.createQuery(query, AnalysisContext.class)
-                .setParameter("project", project)
-                .getResultList();
+        TypedQuery q = this.entityManager.createQuery(query, AnalysisContext.class);
+        q.setParameter("project", project);
+        return q.getResultList();
```

Comment:
My ignorance, are these changes needed for the mocked [entityManager](https://github.com/windup/windup-web/pull/562/files#diff-c8e1b2cec7de438f28267275ac960be3R39) to work?

Reply:
I think so. Alternatively, you can add "doReturn(mockQuery).when(mockQuery).setParameter("project", project);" to the test itself and this change won't be required.

Reply:
So this change can be reverted changing the test cases to fully mock the queries.

Reply:
@m-brophy sorry for pestering you about this but can you please revert these changes (and the one below)?